### ISSUE TITLE
Error message says application is already running, even though it isn't

### ISSUE THREAD

**Issue by Neurrone** [SigmaNight/basiliskLLM#291]
I get this error on startup

```
basiliskLLM is already running. Use the tray icon to interact 
with the application or AltGr+Shift+B to focus the window.
```

I don't see it in task manager, or in the system tray. I'm not sure where the AltGr key is.

**Comment by Nael-Sayegh** [comment]
Hello,  
The AltGr key is the key to the right of the space bar.  
Could you try the shortcut to see if the main window appears?  
Does this message appear on the first launch or when the application is already running?

**Comment by Neurrone** [comment]
No, the application wasn't launched before this, so I'm not sure what went wrong there.

I restarted my machine, which seems to have fixed the issue.

The shortcut doesn't work for me. I pressed the right alt + shift + B keys.

**Comment by Nael-Sayegh** [comment]
The shortcut only works if the application is running in the background. Was it properly launched or minimized when you tried it? Since the issue is resolved, I suggest we close this case.

**Comment by Neurrone** [comment]
The application was properly launched and already in the background when I tried the shortcut. Nothing happened when I tried pressing right alt, shift and b.

I'll close the issue for now, since it hasn't happened again.

**Comment by AAClause** [comment]
Thanks @Neurrone for the report.
Is it the same for the other global shortcuts?

* AltGr+Shift+W: Take a screenshot of the current window and send the image to the active chat.
* AltGr+Shift+F: Take a screenshot of the full screen and send the image to the active chat.

**Comment by Neurrone** [comment]
Yup, the AltGr key shortcuts don't work for me at all.

**Comment by AAClause** [comment]
Could you try with Alt+Ctrl instead of AltDr please?

**Comment by Neurrone** [comment]
Yup, same result. They don't work for me too. I'm using the standard US-english keyboard layout

**Comment by AAClause** [comment]
Could you please test the following try build with the following global shortcuts and let me know if it works better?

- **Win + Alt + Space**: Minimize or restore the BasiliskLLM window to/from the system tray.
- **Win + Ctrl + U**: Capture a screenshot of the entire screen and send the image to the active chat.
- **Win + Ctrl + W**: Capture a screenshot of the active window and send the image to the currently active chat.

https://github.com/SigmaNight/basiliskLLM/actions/runs/11427106384

I'd like to find better shortcuts, but it's challenging to avoid conflicts.

Thanks!

**Comment by Neurrone** [comment]
How about making the hotkeys configurable by the user?

Win + Alt + Space conflicts with the hotkey that switches keyboard input language for me. But the other two hotkeys you mentioned work for me.

**Comment by AAClause** [comment]
> How about making the hotkeys configurable by the user?

Yes, that would be the best thing to do. I'll look into that soon.

**Comment by fastfinge** [comment]
I also get this error: "basiliskLLM is already running. Use the tray icon to interact 
with the application or AltGr+Shift+B to focus the window."

It isn't running. Task manager doesn't show it running. None of its files are in use. I've rebooted my entire machine, even. It still says it's running when it is not.

**Comment by fastfinge** [comment]
Just for fun, tried completely uninstalling and reinstalling the application, and deleting the files it had in %appdata%. No luck; it still claims it's already running.

**Comment by rperez030** [comment]
I have the same problem in one of my machines. None of the global shortcuts work, and apparently there is no error in the log.

**Comment by Neurrone** [comment]
I just ran into this too again, so there's definitely some sort of bug.

**Comment by Neurrone** [comment]
Is there anything that I can do to help with debugging this? It is happening frequently for me now.

**Comment by Neurrone** [comment]
This started happening again for me recently, not sure if anything was in the last few updates that may have caused this. The only fix I've found is to restart my computer, which is pretty inconvenient.
